detection_rules/kbwrap.py

```diff
@@ -71,6 +72,36 @@ def upload_rule(ctx, rules, replace_id):
     return results
 
 
+@kibana_group.command('import-rules')
+@click.argument('rules', nargs=-1, required=True)
+@click.option('--overwrite', '-o', is_flag=True, help='Overwrite existing rules')
```

Comment:
Note: This is needed if customers are testing rules in Dev `space` and then try to migrate to a Prod `space`.